### ISSUE TITLE
Support for timezones?

### ISSUE THREAD

**Issue by larskruckow** [offen/docker-volume-backup#28]
I can see that it runs with default UTC time.
This makes it difficult to time everything to specific local times (especially with daylight saving)
I suggest adding support for setting the timezone it runs under in the environment

**Comment by calebdjgreen** [comment]
This solution does not work any longer due to localtime being a symbolic link in newer distributions.

Can we really not afford the extra 3MB to make this work across all platforms? Seems silly to me.

See https://serverfault.com/questions/683605/docker-container-time-timezone-will-not-reflect-changes for more info and proposed solutions.

**Comment by calebdjgreen** [comment]
This solution does not work any longer due to localtime being a symbolic link in newer distributions.

Adding this along with the other two lines fixed it for me, but this may not always be required:
```
- /usr/share/zoneinfo:/usr/share/zoneinfo:ro
```

Seems like adding the tzdata package may be easier in the long run for simplicities' sake, but I understand not wanting to bloat the image.

See https://serverfault.com/questions/683605/docker-container-time-timezone-will-not-reflect-changes for more info and proposed solutions.

**Comment by mmichalak-swe** [comment]
@m90 This is a wonderful tool you've built, and I think it would greatly benefit from the `tzdata` package included in the image by default.

I did some testing and would like to share the results, both custom images were created with the following Dockerfile:

```Dockerfile
FROM offen/docker-volume-backup:v2.47.2
RUN apk add tzdata --no-cache
```

|architecture|image tag|size (uncompressed on host)|total vulnerabilities in trivy scan|trivy scan breakdown|
|---|---|---|---|---|
|linux/arm64|v2.47.2|42.8 MB|7|**offen/docker-volume-backup:v2.47.2 (alpine 3.23.3)** (alpine)<br>Total: 2 — UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 0, CRITICAL: 1<br><br>**usr/bin/backup** (gobinary)<br>Total: 5 — UNKNOWN: 0, LOW: 2, MEDIUM: 3, HIGH: 0, CRITICAL: 0|
|linux/arm64|v2.47.2-custom-tzdata|43.3 MB|7|**83dd259d5f28 (alpine 3.23.3)** (alpine)<br>Total: 2 — UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 0, CRITICAL: 1<br><br>**usr/bin/backup** (gobinary)<br>Total: 5 — UNKNOWN: 0, LOW: 2, MEDIUM: 3, HIGH: 0, CRITICAL: 0|
|linux/amd64|v2.47.2|45.1 MB|7|**offen/docker-volume-backup:v2.47.2 (alpine 3.23.3)** (alpine)<br>Total: 2 — UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 0, CRITICAL: 1<br><br>**usr/bin/backup** (gobinary)<br>Total: 5 — UNKNOWN: 0, LOW: 2, MEDIUM: 3, HIGH: 0, CRITICAL: 0|
|linux/amd64|v2.47.2-custom-tzdata|45.6 MB|7|**d33369e9f755 (alpine 3.23.3)** (alpine)<br>Total: 2 — UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 0, CRITICAL: 1<br><br>**usr/bin/backup** (gobinary)<br>Total: 5 — UNKNOWN: 0, LOW: 2, MEDIUM: 3, HIGH: 0, CRITICAL: 0|

A difference of ~500kB when the image is uncompressed is negligible, with the added utility far outweighing the cost of a slightly larger image (less than 1.5% larger when uncompressed, even less of a difference when compressed), with no increase in vulnerabilities with the addition of the added package. The documentation could be simplified, and usage overall more intuitive, and in-line with other popular container images. I run a number of popular apps and this is a first for me, all the others allow for timezone being set by a variable (usually TZ).

Thank you for all of your work on this project, it really is a fantastic tool.

**Comment by m90** [comment]
Setting `TZ` is now supported as of [v2.48.0](This is now released in [v2.48.0](https://github.com/offen/docker-volume-backup/releases/tag/v2.48.0))
